src/components/LeftPane/Drawers/AnalyticsDrawer/ServiceAreaAnalytics.tsx

```diff
@@ -1,11 +1,41 @@
 import * as React from 'react'
 import { withStore } from '../../../../services/store'
+import { averageDistance, averageTime, maxDistance, maxTime, minDistance, minTime } from '../../../../utils/analytics'
```

Comment:
I am surprised by the relative paths used, is that common practice?

Reply:
Yeah. It's a little ugly, and may be a sign that we should flatten the folder structure for LeftPane/Drawers.